lib.rs

```diff
@@ -195,6 +183,51 @@ mod chocolate {
                 }
             }
         }
+
+        #[ink(message)]
+        pub fn initiate_verfication_flow(&mut self) -> Result<Vec<u8>> {
+            // Cannot re-initiate flow if already begun
+            if self
+                .account_verification_flow_initiation
+                .contains(&self.env().caller())
+                {
+                    return Err(Error::VerificationFlowAlreadyInitiated);
+                }
+            // Increment verification flow for uniqueness
+            self.verifications_count = self.verifications_count.saturating_add(1);
+
+            // Combine account and id for unique signable message
+            let mut verification_message = self.env().caller().encode(&mut Vec::new());
+            todo!("Add verifications_count to verification message");
+            Ok(todo!("Sign verification message"))
```

Comment:
This part has been filled here https://pastebin.com/8YTPj7tH,
 Is there a reason why it's todo?

Reply:
Yeah, some part of the code didn't work, so this is work in progress (WIP)